test/test_assembling.py

```diff
@@ -181,7 +181,7 @@ def func(*file_paths):
     argh.set_default_command(parser, func)
 
     assert parser.add_argument.mock_calls == [
-        mock.call('file_paths', nargs='*',
+        mock.call('file_paths', nargs='*', metavar='file-paths',
```

Comment:
It's always called with a metavar now, see line 120 of the patch to assembling.py.